upup/pkg/fi/cloudup/new_cluster.go

```diff
@@ -233,9 +285,11 @@ func NewCluster(opt *NewClusterOptions, clientset simple.Clientset) (*NewCluster
 	cluster.Spec.IAM = &api.IAMSpec{
 		AllowContainerRegistry: true,
 	}
-	cluster.Spec.Kubelet = &api.KubeletConfigSpec{
-		AnonymousAuth: fi.PtrTo(false),
+
+	if cluster.Spec.Kubelet == nil {
+		cluster.Spec.Kubelet = &api.KubeletConfigSpec{}
```

Comment:
It would be simpler to initialize `cluster.Spec.Kubelet` before the iteration over KubernetesFeatureGates

Reply:
Should be simpler now.